src/ray/object_manager/object_manager.cc

```diff
@@ -779,25 +784,16 @@ bool ObjectManager::ReceiveObjectChunk(const NodeID &node_id, const ObjectID &ob
                  << ", chunk data size: " << data.size()
                  << ", object size: " << data_size;
 
-  bool still_required;
-  if (!pull_manager_->IsObjectActive(object_id, &still_required)) {
-    if (still_required) {
-      num_chunks_received_thrashed_++;
-    } else {
-      num_chunks_received_cancelled_++;
-    }
+  if (!pull_manager_->IsObjectActive(object_id)) {
```

Comment:
Can we keep the thrashed accounting? It's still relevant for non-`ray.get` requests.